CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to
 ## Unreleased
 
 #### Breaking Changes
+- Unreachable code is now an error
```

Comment:
In C this is a warning. Why make it an error here?

Reply:
Sure, I think I can leave it as a warning with some creative transformations.